### PR TITLE
Testbed style

### PR DESCRIPTION
**Pull request by apisit**
Everything excepts a deploy function. I'll make a dedicated page for deploy.

### CODE REVIEW

File: packages/neo/test/index.html [hunk 1]
```diff
@@ -1,377 +1,564 @@
...
-  <title>O3 Dapp Platform - NEO</title>
-
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-  <link rel="stylesheet" href="./styles.css">
-  <!-- <script type="text/javascript" src="../../core/lib/o3-dapi-core.min.js"></script> -->
-  <!-- <script type="text/javascript" src="../lib/o3-dapi-neo.min.js"></script> -->
-  <!-- <script type="text/javascript" src="../../core/build/bundle.js"></script> -->
-  <!-- <script type="text/javascript" src="../build/bundle.js"></script> -->
-
+  <meta name="description" content="O3 Service Status">
```
Comment (nickfujita) [inline packages/neo/test/index.html]: needs to be updated

**Comment by nickfujita** [comment]
Look awesome!

While testing, noticed a couple of things that need updating:

- Update documents link at top to open in a new tab instead of same tab
- Can we move the page title into the navbar, or have some indication of which testbed it is. While scrolling down, there is no reference.
- On first load, the sidebar is open, but if in mobile, this might be annoying? Not sure?

**Comment by apisit** [comment]
- added deploy page.
- updated link
- title to navbar

**Comment by nickfujita** [comment]
Last thing is if you could please change the text color for input fields, as the color is the same as placeholder text for default inputs. So was a bit confusing whether it was a placeholder of actual text input. If you could please make it a plain black or darker, i'm gtg on this.

**Pull request merged**